### PR TITLE
Feature/random pass

### PR DESCRIPTION
**Pull request by pendharevaibhav-cd**
## what
* Added the random password resource to generate authentication token.

## why
* To secure the elasticache clusters with encryption and passowords.
* Community raised this issue to fix authentication token issue.

## references
* https://github.com/clouddrove/terraform-aws-elasticache/issues/56

### CODE REVIEW

**Review by themaniskshah — DISMISSED** [review]
LGTM!

File: variables.tf
Comment (d4kverma) [inline variables.tf:1]: fix unsued variable.

**Review by d4kverma — DISMISSED** [review]
LGTM

**Review by 13archit — APPROVED** [review]
LGTM

**Review by anmolnagpal — APPROVED** [review]
LGTM

**Review by themaniskshah — APPROVED** [review]
LGTM

**Pull request merged**